src/reaction_lammps_mupt/input_parser.py

```
@@ -0,0 +1,103 @@
+- [ ] Enforce: either "Number of monomers" OR "stoichiometric_ratio" must be provided (not both unless you define a rule).
+- [ ] If "stoichiometric_ratio" is given, compute "Number of monomers" based on:
+      - "Number of total atoms" target(s) (likely multiple) and
+      - monomer atom counts / molar masses from RDKit
+- [ ] Derive box size / initial density estimates from monomer counts + density.
+- [ ] Output a clean, consistently-formatted dictionary to feed into main.py.
+"""
+
+from rdkit import Chem
+
+
+from rdkit import Chem
```

Ruff (0.14.14)

[error] Redefinition of unused `Chem` from line 12: `Chem` redefined here

Remove definition: `Chem`

(F811)